### PR TITLE
Connectivity coherence

### PR DESCRIPTION
**Pull request by HosseinShahabi**
Looking forward

### CODE REVIEW

File: toolbox/process/functions/process_corr1n.m [hunk 1]
```diff
@@ -257,7 +259,7 @@ function Test() %#ok<DEFNU>
     bst_process('CallProcess', 'process_snapshot', sTmp, [], ...
         'target',       11, ...  % Connectivity matrix (image)
         'modality',     1, 'orient', 1, 'time', 0, 'contact_time', [-40, 110], 'contact_nimage', 16, ...
-        'Comment',      [sFile.Comment, ': ' sTmp.Comment]);
```
Comment (ftadel) [inline toolbox/process/functions/process_corr1n.m]: Please keep this last modification comment=>Comment

File: toolbox/process/functions/process_cohere1.m [hunk 2]
```diff
@@ -109,11 +118,13 @@
...
-    OPTIONS.CohOverlap    = 0.50;
+    OPTIONS.CohOverlap    = 0.50;  % First pre-define the overlap
     OPTIONS.pThresh       = 0.05;  % sProcess.options.pthresh.Value{1};
-    switch (sProcess.options.cohmeasure.Value)
-        case 1,  OPTIONS.CohMeasure = 'mscohere';
-        case 2,  OPTIONS.CohMeasure = 'icohere';
+    OPTIONS.CohMeasure    = sProcess.options.cohmeasure.Value; 
+
+    % Change the overlap if it is specified
+    if exist('sProcess.options.overlap.Value{1}')
```
Comment (ftadel) [inline toolbox/process/functions/process_cohere1.m]: Replace with:
if isfield(sProcess.options, 'overlap') && isfield(sProcess.options.overlap, 'Value') && iscell(sProcess.options.overlap.Value) && ~isempty(sProcess.options.overlap.Value) && ~isempty(sProcess.options.overlap.Value{1})

Matlab's exist is not mean to test the existence of subfields like this.

To change in other files where this appears

**Comment by ftadel** [comment]
@HosseinShahabi Do you need more time before this PR is merged?
(after fixing the options testing)

**Comment by ftadel** [comment]
?

**Comment by HosseinShahabi** [comment]
So sorry @ftadel for the late reply. It is now fixed and you can merge it.

**Pull request merged**